script.js

```diff
@@ -65,270 +37,158 @@ while (playing) {
   }
 
 
-  //METHODS
-
-  // //Detects if disk is let go over another tower
-  // function detectLocation(disk, moveCount) {
-  //   //if drop it on one of the other two towers, call move(), passing that tower among other vars
-  //   $(`tower${disk.tower + 1 % 3}`).on("mouseover", move(disk, moveCount, `tower${disk.tower + 1 % 3}`));
-  //   $(`tower${disk.tower + 2 % 3}`).on("mouseover", move(disk, moveCount, `tower${disk.tower + 2 % 3}`));
-  // } // if mouse is let go over another tower, call move() function with disk and tower
-
-  //
-  // //determines whether you can move that disk to the tower you're dropping it on
-  // function move(disk, moveCount, destinationTower) {
-  //   if (destinationTower.contents.length === 0) {
-  //     // destinationTower.contents.push(disk.order); //push disk.order to tower's contents
-  //     moveToTower(disk, moveCount, destinationTower); //destinationTower's contents is empty
-  //     //so move disk there
-  //   } else if (disk.order > destinationTower.contents[destinationTower.contents.length - 1]) { //if trying
-  //     //to place a disk on smaller disk
-  //     console.error("Invalid move: you can only stack disks in increasing size order");
-  //   } else { //move is valid
-  //     moveToTower(disk, moveCount, destinationTower); //move disk to new tower with moveToTower()
-  //   }
-  // }
-  //
-  // //moves the disk to tower, updates disk.tower, and increments moveCount
-  // function moveToTower(disk, destinationTower, moveCount) {
-  //   destinationTower.contents.push(disk.order); //push disk to destinationTower's contents array
-  //   disk.tower = destinationTower.number; // set disks tower number to match destinationTower
-  //   moveCount++; //increment moveCount to indicate another move
-  // }
-
   //picks a random color for disk
   function getRandomColor() {
     var values = "0123456789ABCDEF";
     var hash = "#";
-    for (var i = 0; i < 6; i++) {
+    for (var i = 0; i < 6; i++) { // 6 is an example of a 'magic number'
+    // Better practice is to have a constant like...
+    // const HEX_DIGITS = 6
+    // and then use it in a for-loop like this...
+    // for (i = 0; i < HEX_DIGITS; i++) {
       hash += values[Math.floor(Math.random() * 16)];
     }
     return hash;
   }
 
-  $(init);
+
+  $(init); // would just use init() to call init, especially since init doesn't
+  // return anything
 
   function init() {
+    // You can replace your for-loops with https://api.jquery.com/each/
     for (var i = 0; i < 3; i++) {
-      $(`<div class="tower" id="tower${i}"></>`).data("number", i).appendTo(".playingField");
-      $(`<div class="rod" id="rod${i}"></>`).data("number", i).appendTo(`#tower${i}`);
-      $(`<div class="base" id="base${i}"></>`).appendTo(`#tower${i}`);
+      // since you have a container for your rods, take a look at the link below
+      // https://api.jquery.com/event.stoppropagation/
```

Comment:
@superbuggy would I be using stopPropagation() here? It looks like you only use that with events